.env.example

```diff
@@ -0,0 +1,3 @@
+# AUTH_MODE
+# skips authenticating GraphQL requests with Auth0 when set to "SKIP"
+AUTH_MODE=SKIP
```

Comment:
💁🏻 I'm open to other thoughts on this but for now skipping auth for GraphQL endpoints by default seemed unobtrusive while get the full permissions story sorted out

Reply:
Seems good to me 👍